lib/helpers/mail/Mail.php

```diff
@@ -758,6 +805,9 @@ public function getType()
     public function setValue($value)
     {
         $this->value = mb_convert_encoding($value, 'UTF-8', 'UTF-8');
+        if ($this->type === 'text/plain') {
```

Comment:
I don't think we should call this here, instead I propose we document this function in USE_CASES.md and the developer can optionally use this functionality when desired. That way we don't introduce new unexpected behavior for current users.